src/main/java/duke/Task.java

```diff
@@ -0,0 +1,94 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Represents a task entered by the user.
+ */
+public class Task {
+    private String name;
+    private Boolean done;
```

Comment:
Maybe add a prefix such as `is` before `done` such that the variable sounds more "boolean like"?